db/migrate/20240120174625_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.text :Title
+      t.text :Text
+      t.integer :ComentsCounter
+      t.string :LikesCounter
+
```

Comment:
- [ ] Good job so far, please check that the in posts table the column holding the foreign key to the users table should be named author_id  `t.references :author, foreign_key: { to_table: 'users' }`

- This will create author_id and a corresponding index you can  add column and index in separate commands

Reply:
Hello Prolajumokeoni
thanks for your feedback.
Actually, I did that in the second branch. so when I merge them they will be in the correct way. is it ok to keep on 
here is the link for the PR where I did the changes you required 
https://github.com/AlaaAlsalem/blogaapp/pull/4

Reply:
- Good job, please update the link to your MIT license page it is broken, just realized I didn't request changes on your pull readme 👍🏿